core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -179,6 +179,18 @@ package object config {
   private[spark] val EVENT_LOG_CALLSITE_LONG_FORM =
     ConfigBuilder("spark.eventLog.longForm.enabled").booleanConf.createWithDefault(false)
 
+  private[spark] val EVENT_LOG_ENABLE_ROLLING =
+    ConfigBuilder("spark.eventLog.logRolling.enabled")
+      .doc("Whether rolling over event log files is enabled.")
+      .booleanConf
+      .createWithDefault(false)
+
+  private[spark] val EVENT_LOG_ROLLED_EVENT_LOG_MAX_FILE_SIZE =
+    ConfigBuilder("spark.eventLog.logRolling.maxFileSize")
+      .doc("The max size of event log file to be rolled over.")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("128m")
```

Comment:
I'm checking with configuration.md and it only misses one sentence (`If set to true, it cuts down each event log file to be configured size.`) in above conf. Are you referring other one? Otherwise I'll just add this sentence to doc.

Reply:
> it only misses one sentence (If set to true, it cuts down each event log file to be configured size.) in above conf. Are you referring other one?

Yeah, thanks for the change.